### PR TITLE
ci: add docs format checking

### PR DESCRIPTION
**Pull request by zepatrik**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -154,14 +164,14 @@ workflows:
...
           filters:
             tags:
               only: /.*/
       - nancy/test:
-        filters:
-          tags:
-            only: /.*/
+          filters:
+            tags:
+              only: /.*/
```
Comment (zepatrik) [inline .circleci/config.yml:174]: Noticed that the indentation was wrong here because of validation errors reported by goland

**Comment by zepatrik** [comment]
Looks like there is a problem with `oryd/xgoreleaser:latest` image @aeneasr ? I could not pull it locally either and dockerhub shows a size of 0B https://hub.docker.com/r/oryd/xgoreleaser/tags
Just wondering why the latest master build was working??

**Review by aeneasr — APPROVED** [review]

**Comment by aeneasr** [comment]
@zepatrik validation fails 🤣

**Comment by aeneasr** [comment]
Still failing :/

**Comment by zepatrik** [comment]
Damn, our install script always uses the latest prettier tag, and that behaves differently than our version in this PR...
I think it will be best to use `jq` to get and install the version from `package.json`

**Comment by zepatrik** [comment]
Will hopefully work now...

**Pull request merged**